clientLibrary/README.md

```diff
@@ -0,0 +1,48 @@
+# Login-service client library
+
+This library provides client-functionality for the login-service.
+
+## Usage
+
+Include the library in your project:
+TODO: add maven/SBT dependency
```

Comment:
Hi

So made the changes. However, haven't added the mavencentral badge yet because I can't find the library in Maven Central. I can find it in the nexus but can't see it in Maven Central. Not sure if it's because it's a snapshot at the moment or if there is another step to publishing required?

Reply:
The badge we can add later of course -- when actually released